### PR TITLE
Doomslayer Fixes/Balance

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
The doom chainsaw now blocks ranged attacks, and has some synpatazine for even more anti stun.

It also automatically turns itself on when picked up, and automatically wields

### CODE REVIEW

File: code/game/objects/effects/mines.dm [hunk 1]
```diff
@@ -133,8 +133,10 @@
 	victim.drop_r_hand()
 	victim.drop_l_hand()
 	victim.put_in_hands(chainsaw)
-
+	chainsaw.attack_self(victim)
+	chainsaw.wield(victim)
 	victim.reagents.add_reagent("adminordrazine",25)
+	victim.reagents.add_reagent("synaptizine",25)
```
Comment (Fox-McCloud) [inline code/game/objects/effects/mines.dm]: This is literally pointless when adminordrazine already sets your weakened, paralysis, and stunned to 0---the only thing this is accomplishing is giving a 30% chance to do 1 tox damage, which will be healed the very next time adminordrazine processes

Comment (KorPhaeron) [inline code/game/objects/effects/mines.dm]: whoops!

File: code/game/objects/items/weapons/twohanded.dm [hunk 2]
```diff
@@ -435,11 +436,18 @@
...
-
 /obj/item/weapon/twohanded/required/chainsaw/doomslayer
 	name = "OOOH BABY"
 	desc = "<span class='warning'>VRRRRRRR!!!</span>"
 	armour_penetration = 100
+	force_on = 30
+
+/obj/item/weapon/twohanded/required/chainsaw/doomslayer/hit_reaction(mob/living/carbon/human/owner, attack_text, final_block_chance, damage, attack_type)
+	if(attack_type == PROJECTILE_ATTACK)
+		owner.visible_message("<span class='danger'>Ranged attacks just make [owner] angrier!</span>")
```
Comment (GunHog) [inline code/game/objects/items/weapons/twohanded.dm]: Add a message to the owner himself in BIG RED TEXT saying how much ANGRIER they are! (The spam serves to actually anger the player)

**Comment by optimumtact** [comment]
Finish soon pls

**Comment by optimumtact** [comment]
I love u

**Pull request merged**